### DISCUSSION TITLE
Highlight legend value that exceeds threshold

### DISCUSSION THREAD

**Discussion by kevisong** [grafana/grafana#38068] [Dashboards, Panels & Visualizations]
When using time series with specified threshold. Would it be better to highlight the legend value that exceeds the given threshold?

For example, specify panel threshold as below:

![wecom-temp-788ba442b1fc2ed560f5e30b54dcd2b2](https://user-images.githubusercontent.com/39701004/130065597-5cc331db-097f-4255-ac4f-eede1362e76f.png)

And the first legend value (35.5%) that exceeds given threshold could be rendered as red corresponding to the threshold specified above.

![image](https://user-images.githubusercontent.com/39701004/130065694-e17f64c2-ac95-4769-8fc1-989ec981c0cc.png)

It will be nice and clear which legend value exceeds the threshold.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
